src/Build/Microsoft.Build.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(LibraryTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(RuntimeOutputTargetFrameworks)</TargetFrameworks>
```

Comment:
Why does this require going from netstandard to netcoreapp?

Reply:
Ah, found the reason in the commit message :)

---
In reply to: [176602885](https://github.com/Microsoft/msbuild/pull/3106#discussion_r176602885) [](ancestors = 176602885)